src/cljs/athens/views/blocks/content.cljs

```diff
@@ -316,6 +317,39 @@
 
 
 ;; View
+;; Handlers will come from textarea-keydown
+(defn content-keybindings
+  [uid state child]
+  (let [event-wrapper
+        ;; Only handle event if the currently editing uid is the same
+        ;; as this block; and there aren't any selected item.
+        (fn [callback-or-config _]
+          (let [wrap-callback
+                (fn [callback]
+                  (fn [event]
+                    (when
+                      (and
+                        (= uid @(rf/subscribe [:editing/uid]))
```

Comment:
Please don't `deref` inside the helper like that.
Subscribe in `let` and so on.